appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java

```diff
@@ -1273,7 +1275,7 @@ private boolean writeAndroidManifest(File manifestFile) {
 
         // Companion display a splash screen... define it's activity here
         if (isMain && isForCompanion) {
-          out.write("    <activity android:name=\"com.google.appinventor.components.runtime.SplashActivity\" android:screenOrientation=\"behind\" android:configChanges=\"keyboardHidden|orientation\">\n");
+          out.write("    <activity android:name=\"com.google.appinventor.components.runtime.SplashActivity\" android:exported=\"true\" android:screenOrientation=\"behind\" android:configChanges=\"keyboardHidden|orientation\">\n");
```

Comment:
I wonder if export="true" make sense here. I think we would only want the companion app itself to launch this activity, in which case false would be more approprirate.

Reply:
@jisqyv Please review my comment and let me know what you'd prefer to do here. From my point of view, other apps should not be able to launch this activity so setting exported to `false` makes more sense to me.